principles/index.html

```diff
@@ -133,6 +133,25 @@ <h4>Researchers, regulators and auditors should be able to investigate how a sys
     <section>
       <h4>When abuse happens, there must be a mechanism to identify the abuse, limit further access and enable consequences.</h4>
     </section>
+    <section id="comprehensibility">
+      <h3>Comprehensibility</h3>
+      <section>
+        <h4>Users should be able to understand the implications of their participation in advertising systems.</h4>
+        <p>Users won't typically understand the mathematical guarantees of differential privacy or related concepts, but should still be able to see the implications of participation, including:</p>
+          <ul>
+            <li>the capabilities and what different advertising participants may learn about them;</li> 
+            <li>the likely, and unlikely, risks, including identification, disclosure and manipulation;</li> 
+            <li>whom they are trusting to protect their privacy.</li>
+          </ul>
+        <p>Comprehensibility doesn't require that every user understands or has a correct mental model, but that an interested user can reasonably learn and can make informed decisions about participation.</p>
```

Comment:
This is a bit tricky. How about?
```suggestion
        <p>Comprehensibility doesn't require that every user understands or has a correct mental model. An interested user should be able to learn and form an understanding that is accurate enough to predict the consequences of actions such that they could make informed choices about their participation in the system.</p>
```

Reply:
```suggestion
        <p>Comprehensibility doesn't require that every user understands or has a correct mental model. An interested user should be able to learn enough to predict the implications such that they can make informed decisions about participation.</p>
```